packages/twenty-front/package.json

```diff
@@ -32,7 +32,7 @@
     "storybook:test": "test-storybook",
     "storybook:test-slow": "test-storybook --maxWorkers=3",
     "storybook:test-single-worker": "test-storybook --maxWorkers=1",
-    "storybook:coverage": "yarn storybook:test-slow --coverage && npx nyc report --reporter=lcov -t coverage/storybook --report-dir coverage/storybook --check-coverage",
+    "storybook:coverage": "yarn storybook:test-slow --coverage ; npx nyc report --reporter=lcov -t coverage/storybook --report-dir coverage/storybook --check-coverage",
```

Comment:
We want to always collect coverage even if some tests fail or coverage is below the required threshold. The second command after && will only run if the first has a 0 return value, we change this to `;` to ensure the report is always generated.